tests/test_release.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from pytest_mock import MockerFixture
+
+import release
+
+
+@pytest.mark.parametrize(
+    ["test_editor", "expected"],
```

Comment:
Any reason why this is a list and not a string or a tuple?
The docs seem to prefer `"test_editor,expected"`, and a tuple seem to better mirror the tuples in the following list.
```suggestion
    "test_editor,expected",
```
```suggestion
    ("test_editor", "expected"),
```

Reply:
I copied the style of the existing tests: https://github.com/python/release-tools/blob/24bb55e747e6c5517dd4a7943ac35ef1f38a33d7/tests/test_sbom.py#L11-L21